### PR TITLE
chore: bump version to 10.2.0-rc.0 w/ changelog

### PR DESCRIPTION
**Pull request by jelbourn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,130 @@
+# 10.2.0-rc.0 "cotton-cactus" (2020-08-26)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| feature |  add support for TypeScript 4.0 ([#20368](https://github.com/angular/components/issues/20368)) ([f33210c](https://github.com/angular/components/commit/f33210c)) |
```
Comment (andrewseguin) [inline CHANGELOG.md:7]: Should this be part of highlights?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,130 @@
...
+### material
+
+_Deprecations:_
+
+* **autocomplete:** * `AUTOCOMPLETE_OPTION_HEIGHT` is deprecated, because it isn't being used anymore.
+* `AUTOCOMPLETE_PANEL_HEIGHT` is deprecated, because it isn't being used anymore.
+
+|            |                       |
+| ---------- | --------------------- |
+| feature |  add support for TypeScript 4.0 ([#20368](https://github.com/angular/components/issues/20368)) ([f33210c](https://github.com/angular/components/commit/f33210c)) |
```
Comment (andrewseguin) [inline CHANGELOG.md:30]: This shows up twice

Comment (jelbourn) [reply CHANGELOG.md:30]: I intentionally added it twice since it affects both

File: CHANGELOG.md (hunk 1 above)
Comment (jelbourn) [reply CHANGELOG.md:7]: I'll add a highlights section for the stable release

**Pull request merged**